src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx

```diff
@@ -30,7 +37,17 @@ const VaPrescription = prescription => {
             )}
             <h2 className="vads-u-margin-y--2">About your prescription</h2>
             {prescription && <ExtraDetails {...prescription} />}
-            <FillRefillButton {...prescription} />
+            {showRefillContent && prescription?.isRefillable ? (
+              <a
+                className="vads-u-display--block vads-c-action-link--green vads-u-margin-top--3 vads-u-margin-bottom--3"
+                href={medicationsUrls.MEDICATIONS_REFILL}
+                data-testid="refill-nav-link"
```

Comment:
Should we add a test for this new data-testid?

Also just curious, was there a specific reason to use `<a>` instead of `<Link>` here?

Reply:
changing <a> to <Link> now. as for the unit test - I do not see how would it be of a value: the link is not dynamic and such functionality is usually covered by e2e (+ unit test snapshot). if you were referring to feature toggle condition - we are planning to clean it anyway

thank you for your comment btw!!!